plugins/woocommerce/client/legacy/js/frontend/woocommerce.js

```diff
@@ -225,17 +225,19 @@ function focus_populate_live_region() {
  */
 function refresh_sorted_by_live_region() {
 	var sorted_by_live_region = document.querySelector(
-		'.woocommerce-result-count[data-is-sorted-by="true"]'
+		'.woocommerce-result-count'
 	);
 
 	if ( sorted_by_live_region ) {
 		var text = sorted_by_live_region.innerHTML;
-
+		sorted_by_live_region.setAttribute('aria-hidden', 'true');
+		
 		var sorted_by_live_region_id = setTimeout( function () {
+			sorted_by_live_region.setAttribute('aria-hidden', 'false');
```

Comment:
I don't think I understand setting the `aria-hidden` to `true` and changing it to `false` after 2s, do you mind explaining it?

Reply:
@kmanijak When the page loads, screen readers get busy announcing several messages and sometimes the live region is overlooked. Adding the `aria-hidden="true"` makes the screen reader ignore the live region for 2 seconds. This trick ensures that the live region will be announced once `aria-hidden` is set to `false` and its content changes.